synth.py

```diff
@@ -32,18 +32,22 @@
     library = gapic.py_library(
         service="firestore",
         version=version,
-        proto_path=f"google/firestore/{version}",
-        generator_version="v0.26.5"
+        bazel_target=f"//google/firestore/{version}:firestore-{version}-py",
     )
 
     s.move(
-        library / f"google/firestore_{version}",
+        library / f"google/cloud/firestore_{version}",
         f"google/cloud/firestore_{version}",
-        excludes=[ library / f"google/firestore_{version}/__init__.py"]
+        excludes=[
+            library / f"google/firestore_{version}/__init__.py"
```

Comment:
Does this need to include `/cloud`?  E.g.:
```suggestion
            library / f"google/cloud/firestore_{version}/__init__.py"
```